models/ssh_key.go

```diff
@@ -473,11 +473,11 @@ func ListPublicKeys(uid int64) ([]*PublicKey, error) {
 		Find(&keys)
 }
 
-// ListPublicLdapSSHKeys returns a list of synchronized public ldap ssh keys belongs to given user.
-func ListPublicLdapSSHKeys(uid int64) ([]*PublicKey, error) {
+// ListPublicLdapSSHKeys returns a list of synchronized public ldap ssh keys belongs to given user and ldap source.
+func ListPublicLdapSSHKeys(uid int64, lsid int64) ([]*PublicKey, error) {
```

Comment:
please use argument names that makes sense without having to read the rest of the code :)